client.go

```diff
@@ -110,6 +110,10 @@ func (c *Client) Store(req Request, dest string) error {
 	if err != nil {
 		return err
 	}
+	// Check for 2XX Status Codes
+	if resp.StatusCode < http.StatusOK || resp.StatusCode >= http.StatusMultipleChoices {
+		return errors.New("cannot convert file to PDF")
```

Comment:
`couldn't` seems to be better than `cannot` as we do not know if the file can never be converted.